clang/lib/Sema/SemaCodeComplete.cpp

```diff
@@ -6143,6 +6144,13 @@ static FunctionProtoTypeLoc GetPrototypeLoc(Expr *Fn) {
     }
   }
 
+  if (const auto *ME = dyn_cast<MemberExpr>(Fn)) {
```

Comment:
nit: make this an `else if` (i.e. make it part of the chain just above), and then the `!Target` condition below becomes unnecessary

Reply:
Done.